csrankings-b.csv

```diff
@@ -70,7 +70,9 @@ Baoliu Ye,Nanjing University,https://cs.nju.edu.cn/yebl,BlbNI_sAAAAJ
 Baoquan Chen,Peking University,https://cfcs.pku.edu.cn/baoquan,iHWtrEAAAAAJ
 Baotian Hu,Harbin Institute of Technology,http://faculty.hitsz.edu.cn/hubaotian,5NiJ1VoAAAAJ
 Baowen Xu,Nanjing University,https://cs.nju.edu.cn/xubaowen,NOSCHOLARPAGE
+Baoxiang Wang 0001,CUHK-SZ,bxiangwang.github.io,cQe4OeYAAAAJ

```

Comment:
Home page needs to be a properly formed URL.

Reply:
The URL is completed in the newest modification.